cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/transforms/DBRecordToByteArrayTransform.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.templates.etl.transforms;
+
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.cdap.templates.etl.api.Emitter;
+import co.cask.cdap.templates.etl.api.Transform;
+import co.cask.cdap.templates.etl.common.DBRecord;
+import com.google.gson.Gson;
+import org.apache.hadoop.io.LongWritable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+
+/**
+ * Transforms a record from a database table into a byte array containing the json representation of the record
+ */
+public class DBRecordToByteArrayTransform extends Transform<LongWritable, DBRecord, byte [], byte []> {
```

Comment:
Would it be better to have two separate Transforms:
DBRecord to StructuredRecord
StructuredRecordToByteArray

So that we can have other transforms like FilterTransform, ProjectionTransform inserted between the above two transforms and these transforms work on StructuredRecords.